spec/controllers/decidim/action_delegator/admin/delegations_controller_spec.rb

```diff
@@ -18,7 +18,28 @@ module Admin
           sign_in user
         end
 
+        describe "#index" do
+          it "authorizes the action" do
+            expect(controller).to receive(:allowed_to?).with(:index, :delegation, {})
+
+            get :index
+          end
+
+          it "renders the index template" do
+            get :index
+
+            expect(response).to render_template(:index)
+            expect(response).to have_http_status(:ok)
+          end
+        end
+
         describe "#destroy" do
+          it "authorizes the action" do
+            expect(controller).to receive(:allowed_to?).with(:destroy, :delegation, {})
```

Comment:
*(following from previous suggestion)*

...and 

```suggestion
            expect(controller.allowed_to? :destroy, :delegation).to eq true
```